keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -664,6 +672,13 @@ endpoints:
 The `ServiceImport.Spec.IP` (VIP) can be used to access this service from within
 this cluster.
 
+
+#### Multi-network scenario
+One of more clusters in a ClusterSet can be running on a discrete network (a non-flat network). An MCS controller can use the `network.k8s.io` `ClusterProperty` to determine if a cluster in a `ClusterSet` is running on a discrete network. In this scenario, the ServiceImport and the EndpointSlice can be generated to point to the Gateway/Loadbalancer of destination cluster. Note that the endpoints of the `EndpointSlice`s may only be representative of the pods backing the multi-cluster service and not real pod addresses, and may actually represent multiple pods. The MCS controller implementation is responsible for determining the Gateway/Loadbalancer in a cluster for a non-flat network. An MCS controller implementation can use a K8s [Gateway](https://gateway-api.sigs.k8s.io/) defined with an annotation `network.k8s.io: mcs-gateway` to match against a ServiceExport with the same annotation to determine the Gateway to be used. 
```

Comment:
I don't understand the point about a gateway annotation?

Reply:
I'm not sure we need to call out the gateway annotation. In a previous comment I suggested we trim to just recommending that a standard cluster property (`network.k8s.io` or other) is used for the cluster's network. Really, that's more to help fix the lack of cluster self-awareness than to really enable implementations. If there's other work going on there it would be best not to duplicate. @thockin how can I get plugged in? multi-net and multi-cluster have a lot of overlap.

I believe this spec already provides the flexibility required for an implementation to do whatever it needs to for multi network - if we need to more explicitly highlight that the IPs in EndpointSlices don't have to be real IPs from another cluster we should start there.

Reply:
@JeremyOT I removed the gateway references, can I keep the PR until we find the details about the ongoing effort?
@thockin Thanks for the feedback, would love to learn more about the ongoing effort.

Reply:
@thockin Can you please let me know about the ongoing effort?

Reply:
multi-net exists in the form of `multus` and there have been some new mumbles about making it more first-class.  I think it's probably safe  to assume that there's a "default' network or "pod network", and this property models that?

Reply:
Yep, thats right.